gwpy/segments/flag.py

```diff
@@ -345,13 +345,13 @@ def padding(self):
     @padding.setter
     def padding(self, pad):
         if pad is None:
-            self._padding = (0, 0)
+            self._padding = (0., 0.)
         else:
-            self._padding = (pad[0], pad[1])
+            self._padding = tuple([0. if p is None else float(p) for p in pad])
```

Comment:
@tjma12, this can be simplified to

```python
    def padding(self, pad):
        if pad is None:
            pad = (None, None)
        self._padding = tuple(float(p or 0.) for p in pad)
```

can you (test and) make that change?